.ci/bump-golang.yml

```diff
@@ -0,0 +1,137 @@
+---
+name: Bump golang-version to latest version
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: beats
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+actions:
+  beats:
+    kind: github/pullrequest
+    scmid: githubConfig
+    sourceid: latestGoVersion
+    spec:
+      automerge: false
+      labels:
+        - dependencies
+        - backport-skip
+      title: '[Automation] Bump Golang version to {{ source "latestGoVersion" }}'
+      description: |
+        It requires the version to be bumped first in golang-crossbuild project, then a new release will be added to:
+        https://github.com/elastic/golang-crossbuild/releases/tag/v{{ source "latestGoVersion" }}.
+        Otherwise it will fail until the docker images are available.
+
+sources:
+  minor:
+    name: Get minor version in .go-version
+    kind: shell
+    transformers:
+      - findsubmatch:
+          pattern: '^\d+.(\d+).\d+$'
+          captureindex: 1
+    spec:
+      command: cat .go-version
```

Comment:
This file is the one driving the bump for the patch releases. Hence if a new minor is needed then it will require the change to be done in this file first.